File: csharp/DVSClient/DVSClient/server/address/search/RestApiAddressSearchResponse.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Search
{
    public class RestApiAddressSearchResponse : RestApiResponse
    {
        [JsonProperty("error")]
        public RestApiResponseError? Error { get; set; }

        [JsonProperty("result")]
        public RestApiAddressSearchResult? Result { get; set; }
    }
}

File: typescript/src/server/address/search/restApiAddressSearchResponse.ts
import { RestApiResponse } from "../../restApiResponse";
import { RestApiResponseError } from "../../restApiResponseError"
import { RestApiAddressSearchResult } from "./restApiAddressSearchResult";

export type RestApiAddressSearchResponse = RestApiResponse & {
    error?: RestApiResponseError;
    result?: RestApiAddressSearchResult;
}